samples/bot-calling-meeting/csharp/Source/CallingBotSample/Bots/MessageBot.cs

```diff
@@ -256,17 +222,128 @@ private async Task SendResponse(ITurnContext<IMessageActivity> turnContext, stri
+            if (callId == null)
+            {
+                // Without the Meeting ID we are unable to play the prompt
+                //await turnContext.SendActivityAsync("Meeting ID not found, please use the 'Create call' button to create the call.");
+                return await errorHandler("Meeting ID not found, please use the 'Create call' button to create the call.");
+            }
+
+            try
+            {
+                await function();
+            }
+            catch (ServiceException ex)
+            {
+                if (ex.StatusCode != HttpStatusCode.NotFound)
+                {
+                    // If it's not a NotFound error please ignore
+                    throw ex;
+                }
 
-            var resourceResponse = await turnContext.SendActivityAsync(MessageFactory.Attachment(adaptiveCardFactory.CreateMeetingActionsCard()));
-            conversationData.MeetingActionsCardActivityId = resourceResponse.Id;
-            await conversationState.SaveChangesAsync(turnContext, false, cancellationToken);
+                logger.LogError("Call not found. Return error");
+                return await errorHandler("That action failed. Unable to find call");
+            }
+
+            return default;
         }
 
-        private TaskModuleResponse CreateTaskModuleResponse(string value)
+        private async Task<Call> JoinScheduledMeeting(ITurnContext turnContext, CancellationToken cancellationToken)
+        {
+            var users = await TeamsInfo.GetPagedMembersAsync(turnContext, cancellationToken: cancellationToken);
+
+            return await callService.Create(
+                new ChatInfo { ThreadId = turnContext.Activity.Conversation.Id },
+                new OrganizerMeetingInfo
+                {
+                    Organizer = new IdentitySet
+                    {
+                        User = new Identity
+                        {
+                            Id = users.Members[0].AadObjectId,
+                            AdditionalData = new Dictionary<string, object> { { "tenantId", azureAdOptions.TenantId } }
+                        },
+                    },
+                });
+        }
+
+        private async Task<TaskModuleResponse> CreateIncidentCall(ITurnContext turnContext, string incidentSubject, string[] peoplePickerAadIds, CancellationToken cancellationToken)
+        {
+            var onlineMeeting = await onlineMeetingService.Create(incidentSubject, peoplePickerAadIds);
+
+            if (onlineMeeting != null)
+            {
+                (ChatInfo chatInfo, MeetingInfo meetingInfo) = JoinInfo.ParseJoinURL(onlineMeeting.JoinWebUrl);
+
+                var meetingCall = await callService.Create(chatInfo, meetingInfo);
+
+                if (meetingCall != null)
+                {
+                    await chatService.InstallApp(meetingCall.ChatInfo.ThreadId, "60413772-76b9-48c3-b7e5-2ecc82f60f41");
```

Comment:
What's the GUID here?

Reply:
Good catch, meant to replace this with a setting, will do that now.

Reply:
It's the catalog app id, I meant to replace this with a value in appsettings. Will update

Reply:
It's the catalog App Id, I've updated the documentation of the function too include this too